README.md

```diff
@@ -18,10 +18,10 @@ HedgeDoc 2
 HedgeDoc lets you create real-time collaborative markdown notes. Have a look at [our website](https://hedgedoc.org) for
 more details.
 
-**Important:** We are currently working on HedgeDoc 2, a complete rewrite of HedgeDoc. Please note the following:
+⚠️⚠️⚠️ **Important:** We are currently working on HedgeDoc 2, a complete rewrite of HedgeDoc. Please note the following:
```

Comment:
Maybe we should include a warning in the title of the README to make it make as obvious as possible.